navigation.yaml

```diff
@@ -1616,10 +1616,12 @@ ubuntu1604:
 certification:
   title: Hardware
   path: /certified
+  persist: True
+  match_query: True
 
   children:
     - title: Overview
-      path: /certified
+      path: /certified?
     - title: Laptops
       path: /certified/laptops
```

Comment:
I suppose this needs changing to `/certified?category=Laptops`, and all the others? The Laptop, Desktop etc. currently aren't being highlighted.

Reply:
Sorry, I thought it would work after rebasing with the changes made in `redirects.yaml` but it doesn't seem to work that way. Fixed now.